libraries/AP_ESC_Telem/AP_ESC_Telem.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL.h>
+
+class AP_ESC_Telem {
+public:
+
+    AP_ESC_Telem();
+
+    /* Do not allow copies */
+    AP_ESC_Telem(const AP_ESC_Telem &other) = delete;
+    AP_ESC_Telem &operator=(const AP_ESC_Telem&) = delete;
+
+    static AP_ESC_Telem *get_singleton();
+
+    // methods to determine which ESC telemetry types are active
+    bool tcan_is_active() const;
```

Comment:
Do we really need these helpers, or can we build a mask of some form for which ones are available? (Or maybe an enum of possible types where we ask if a specific type is active?). Just seems like a weird thing to expose here.

Reply:
no we don't.  I put them there as an example of what else we might add into AP_ESC_Telem but because they're not used immediately they're breaking one of our golden rules re having unused code around so I've remove these accessors now.